### DISCUSSION TITLE
Chart visualization module for DKAN

### DISCUSSION THREAD

**Discussion by stefan-korn** [GetDKAN/dkan#4147] [Show and tell]
Currently working on visualization for DKAN using [Chartjs](https://www.chartjs.org). Module is in very early stage, but basically working as far as I can tell: https://www.drupal.org/project/dkan_chart

You will just need to enable the module and after that you will find a "tab" called "Visualization" on each dataset node that has a distribution in the datastore. Then you can try out some visualizations. If using the react frontend you may need to edit a dataset to see the "Visualization" tab.

Not really meant for production at this moment, but wanted to let folks know. I will still improve on this.

Happy if someone likes to test and maybe leave a feedback.

**Comment by mgifford** [comment]
Any progress with this? I was just running into some of the accessibility work that's been built into https://www.chartjs.org

**Comment by stefan-korn** [comment]
It kind of works, but probably only tested in our project currently. But should work with DKAN Datastore API without any additional requirement beside DKAN and clipboardjs module.

You can see an example here: https://offenedaten-koeln.de/dataset/einsatzgeschehen-feuerwehr-koeln-2023

I have not ported the module to DKAN 3/4, but this is on my agenda.
